README.md

```diff
@@ -0,0 +1,275 @@
+    $ git clone https://github.com/coecms/esm-pre-industrial
+    $ cd esm-pre-industrial
+
+Run the model:
+
+    $ payu run
+
+Check the output:
+
+    $ ls archive/
+
+
+## Understanding **payu**
+
+**payu** was designed to help users of the NCI system run climate models.
+It was initially created for MOM, but has been adapted for other models,
+including coupled ones.
+
+The aim of **payu** is to make it easy and intuitive to configure and run the models.
+
+**payu** knows certain models and how to run them. Adding more models needs additions to the **payu** sources.
+This will not be part of this document.
+
+### Terms
+
+To understand **payu**, it helps to distinguish certain terms:
+
+-   The **laboratory** is a directory where all parts of the model are kept.
+    It is typically in the user's short directory, usually at `/short/$PROJECT/$USER/<MODEL>`
+-   The **Control Directory** is the directory where the model configuration is
+    kept and from where the model is run.
+-   The **work** directory is where the model will actually be run.
+    It is typically a subdirectory of the Laboratory.
+    Submodels will have their own subdirectories in the work directory, named
+    after their name in the master configuration file.
+    It is ephemeral, that means payu will clean it up after the run.
+-   The **archive** directory is where **payu** puts all output files after each run.
+
+The **work** and **archive** directories will be automatically created by **payu**.
+
+### The master configuration file
+
+In the Control Directory, the file `config.yaml` is the master control file.
+Examples of what is configured in this file are:
+
+-   The actual model to run.
+-   Where to find the model binaries and configurations
+-   What resources to request from the scheduling system (PBS)
+-   Links to the laboratory
+-   Start date and run length per submission pf the model
+
+The model configuration files are typically in subdirectories of the Control Directory,
+the location of which is referenced in the master control file.
+Since the models itself do need different ways to set up the model, the contents of these subdirectories will differ between different models.
+
+## Understanding ACCESS-ESM
+
+ACCESS (Australian Community Climate and Earth System Simulator) is a Coupled Climate Model.
+
+The ESM 1.5 subversion of ACCESS specifically contains these models:
+
+| Component  | Model      | Version |
+| ---------- | ---------- | ------- |
+| Atmosphere | UM-HG3     | 7.3     |
+| Ocean      | MOM        | 4       |
+| Sea Ice    | CICE       | 4.1     |
+| Coupler    | OASIS-MCT  | 3.5     |
+
+Pre-compiled executables for these models are available on raijin at
+`/short/public/access-esm/payu/bin/csiro/`.
+
+## Setting up ACCESS-ESM with **payu**
+
+### The pre-conditions
+
+On `raijin`, first make sure that you have access to our modules.
+This can most easily been done by adding the line
+
+    module use /g/data3/hh5/public/modules
```

Comment:
It is not a good idea to put any module stuff in `~/.bashrc` as it gets sourced every time a new shell is spawned. Either add to `~/.profile` or not at all, and instead make a command to do it, or an alias.